R/get_connection.R

```diff
@@ -88,7 +88,7 @@ get_connection <- function(...) {
         username = cred[1]
         password = cred[2]
       }, error = function(e) {
-        stop("You are not using a Nuvolos application. Please provide both username and password arguments to establish a connection. In your R's console, please add your credentials by this command: input_nuvolos_credential('your_user_name', 'your_password')")
+        stop("You are not using a Nuvolos application. Please provide both username and password arguments to establish a connection. In your R's console, please add your credentials by this command: input_nuvolos_credential()")
```

Comment:
Why not directly call input_nuvolos_credential for the user? Instead of throwing an error?